docs/source/models.rst

```diff
@@ -2317,6 +2317,29 @@ dataframe's column names must match the model signature's column names.
     pyfunc_udf = mlflow.pyfunc.spark_udf(spark, <path-to-model-with-signature>)
     df = spark_df.withColumn("prediction", pyfunc_udf())
 
+If a model contains a signature with tensor spec inputs,
+you need to pass a column of array type as a corresponding UDF argument,
+the column values must be one dimension arrays and the
+UDF will reshape the column values to the required shape with C_CONTIGUOUS (row-major)
```

Comment:
Could we add a link here to the numpy reference for `C_CONTIGUOUS`? Most numpy users would be more familiar with the short-hand notation of `C` and pandas users will likely find this very confusing.

Reply:
I rewrite it to with 'C' order (i.e. read / write the elements using C-like index order) , make it the same as the numpy doc: https://numpy.org/doc/stable/reference/generated/numpy.reshape.html

Reply:
@WeichenXu123 Do we really need `with 'C' order`?

Reply:
I think we should explicitly explain it is using 'C' order , because reshape with order='F' will get different result and some math libraries uses 'F' order by default